src/main/java/com/google/sps/data/ExtendedTask.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.sps.data;
+
+import com.google.api.client.util.GenericData;
+import com.google.api.client.util.Key;
+import com.google.api.services.tasks.model.Task;
+import java.util.Comparator;
+
+// Override equals and hashcode by extending GenericData
+// Two tasks are equal if all of their properties match (It makes testing easier)
+public class ExtendedTask extends GenericData {
```

Comment:
I just wanted to receive a bit of feedback on this solution I used to override Equals and Hashcode. (I'm not sure if this extending-some-class is a usual solution to this, or if it is okay to extend a class that was imported from an API, and not a default java library class.)

Reply:
@dola